modules/blocks/verify.js

```diff
@@ -821,7 +822,9 @@ Verify.prototype.processBlock = function(block, broadcast, saveBlock, cb) {
 			// 'false' if block comes from chain synchronisation process
 			updateSystemHeaders(seriesCb) {
 				// Update our own headers: broadhash and height
-				modules.system.update(seriesCb);
+				!library.config.loading.snapshotRound
```

Comment:
Maybe the opposite?
`library.config.loading.snapshotRound ? seriesCb() : modules.system.update(seriesCb);`

Reply:
I considered that too. But I'd like to provide the desired functionality in the first part and I'd prefer to keep the same structure with rest of the short-hand if usages.